tests/topotests/static_routing_with_ibgp/test_static_routes_topo1_ibgp.py

```diff
@@ -0,0 +1,1313 @@
+        )
+        nh = NEXT_HOP_IP["nh2"][addr_type]
+
+        result = verify_rib(
+            tgen,
+            addr_type,
+            dut,
+            rte2_nh2,
+            next_hop=nh,
+            protocol=protocol,
+            expected=False,
+        )
+        assert result is not True, "Testcase {} : Failed \nError: Routes is"
+        " still present in RIB".format(tc_name)
+
+        nh = [NEXT_HOP_IP["nh1"][addr_type]]
+        result = verify_rib(
+            tgen, addr_type, dut, rte1_nh1, next_hop=nh, protocol=protocol
+        )
+        assert result is True, "Testcase {} : Failed \nError: Routes is"
+        " missing in RIB".format(tc_name)
+
+        step("No shut nexthop interface N2")
+        shutdown_bringup_interface(tgen, dut, intf, True)
+
+        step(
+            "after shut of nexthop N1 , route become active "
+            "with nexthop N2 and vice versa."
+        )
+        rte1_nh1 = {
+            "r2": {
+                "static_routes": [
+                    {
+                        "network": NETWORK2[addr_type],
+                        "next_hop": NEXT_HOP_IP["nh1"][addr_type],
+                        "admin_distance": 10,
+                    }
+                ]
+            }
+        }
+        nh = [NEXT_HOP_IP["nh1"][addr_type]]
+        dut = "r2"
+        protocol = "static"
+        result = verify_rib(
+            tgen, addr_type, dut, rte1_nh1, next_hop=nh, protocol=protocol, fib=True
+        )
+        assert result is True, "Testcase {} : Failed \nError: Routes is"
+        "missing in RIB".format(tc_name)
+
+        rte2_nh2 = {
+            "r2": {
+                "static_routes": [
+                    {
+                        "network": NETWORK2[addr_type],
+                        "next_hop": NEXT_HOP_IP["nh2"][addr_type],
+                        "admin_distance": 20,
+                    }
+                ]
+            }
+        }
+        nh = [NEXT_HOP_IP["nh2"][addr_type]]
+        dut = "r2"
+        protocol = "static"
+        result = verify_rib(
+            tgen,
+            addr_type,
+            dut,
+            rte2_nh2,
+            next_hop=nh,
+            protocol=protocol,
+            fib=True,
+            expected=False,
+        )
+        assert result is not True, "Testcase {} : Failed \nError: Routes is"
+        "not active in RIB".format(tc_name)
+
+        dut = "r3"
+        protocol = "bgp"
+
+        result = verify_rib(
+            tgen,
+            addr_type,
+            dut,
+            rte2_nh2,
+            next_hop=nh,
+            protocol=protocol,
+            fib=True,
+            expected=False,
+        )
+        assert result is not True, "Testcase {} : Failed \nError: Routes is"
+        "not active in RIB".format(tc_name)
+
+        dut = "r2"
+        step("Reload the FRR router")
+        # stop/start -> restart FRR router and verify
+        stop_router(tgen, "r2")
+        sleep(5)
```

Comment:
here too

Reply:
Changes made are:
1. Removed the static sleep.
2. Base config is modified to use bgp aggressive timers.